From dc4541aab2360a65a2207f28b6bd010f650cb0a8 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <b03824@freescale.com>
Date: Wed, 7 Mar 2012 12:03:33 +0800
Subject: [PATCH 1144/2463] ENGR00176068-3 defconfig: enable localtimer in
 defconfig

Enable local timer by default. If wait mode is on,
local timer will be shutdown automatically on boot.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 arch/arm/configs/imx6_defconfig |   10 ++++++++--
 1 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index e3e6d68..726720f 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -389,13 +389,14 @@ CONFIG_GENERIC_CLOCKEVENTS_BUILD=y
 CONFIG_SMP=y
 CONFIG_SMP_ON_UP=y
 CONFIG_HAVE_ARM_SCU=y
+CONFIG_HAVE_ARM_TWD=y
 # CONFIG_VMSPLIT_3G is not set
 CONFIG_VMSPLIT_2G=y
 # CONFIG_VMSPLIT_1G is not set
 CONFIG_PAGE_OFFSET=0x80000000
 CONFIG_NR_CPUS=4
 CONFIG_HOTPLUG_CPU=y
-# CONFIG_LOCAL_TIMERS is not set
+CONFIG_LOCAL_TIMERS=y
 # CONFIG_PREEMPT_NONE is not set
 # CONFIG_PREEMPT_VOLUNTARY is not set
 CONFIG_PREEMPT=y
@@ -1245,6 +1246,7 @@ CONFIG_GPIO_SYSFS=y
 # CONFIG_GPIO_PCA953X_IRQ is not set
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_GPIO_SX150X is not set
+# CONFIG_GPIO_WM8994 is not set
 # CONFIG_GPIO_ADP5588 is not set
 
 #
@@ -1385,6 +1387,7 @@ CONFIG_WATCHDOG_NOWAYOUT=y
 # Watchdog Device Drivers
 #
 # CONFIG_SOFT_WATCHDOG is not set
+# CONFIG_MPCORE_WATCHDOG is not set
 # CONFIG_MAX63XX_WATCHDOG is not set
 CONFIG_IMX2_WDT=y
 
@@ -1457,6 +1460,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 # CONFIG_REGULATOR_MAX8649 is not set
 # CONFIG_REGULATOR_MAX8660 is not set
 # CONFIG_REGULATOR_MAX8952 is not set
+# CONFIG_REGULATOR_WM8994 is not set
 # CONFIG_REGULATOR_LP3971 is not set
 # CONFIG_REGULATOR_LP3972 is not set
 # CONFIG_REGULATOR_MC34708 is not set
@@ -1601,6 +1605,7 @@ CONFIG_MXC_CAMERA_OV5640_MIPI=m
 CONFIG_MXC_CAMERA_OV8820_MIPI=m
 CONFIG_MXC_CAMERA_OV5642=m
 CONFIG_MXC_TVIN_ADV7180=m
+CONFIG_MXC_CAMERA_SENSOR_CLK=m
 CONFIG_MXC_IPU_PRP_VF_SDC=m
 CONFIG_MXC_IPU_PRP_ENC=m
 CONFIG_MXC_IPU_CSI_ENC=m
@@ -1803,12 +1808,13 @@ CONFIG_SND_IMX_SOC=y
 CONFIG_SND_MXC_SOC_MX2=y
 CONFIG_SND_MXC_SOC_SPDIF_DAI=y
 CONFIG_SND_SOC_IMX_SGTL5000=y
+CONFIG_SND_SOC_IMX_WM8958=y
 CONFIG_SND_SOC_IMX_CS42888=y
 CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_IMX_HDMI=y
-CONFIG_SND_SOC_IMX_WM8958=y
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
+CONFIG_SND_SOC_WM_HUBS=y
 CONFIG_SND_SOC_MXC_HDMI=y
 CONFIG_SND_SOC_MXC_SPDIF=y
 CONFIG_SND_SOC_SGTL5000=y
-- 
1.7.7.4

